api/admin/adminRouter.js

```diff
@@ -27,19 +27,19 @@ router.post('/', checkPayload, authRequired, async (req, res, next) => {
     const newadmin = await Admins.addAdmin(req.body);
     res.status(201).json(newadmin);
   } catch (error) {
-    next({ status: 400, message: 'could not create new admin profile' });
+    next({ status: 500, message: 'could not create new admin profile' });
   }
```

Comment:
Nice change from status 400 to 500 to show that the error is from the server and not the user.